packages/back-end/src/services/organizations.ts

```diff
@@ -515,3 +515,25 @@ export async function getExperimentOverrides(organization: string) {
 
   return overrides;
 }
+
+export async function getExperimentTrackingKeys(organization: string) {
```

Comment:
Instead of a separate function, can we just make the existing overrides function return this mapping in addition to the overrides? It will prevent a duplicate database query.

```ts
const { overrides, idMapping } = getExperimentOverrides(...)
```